<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1637</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nyaa Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/images/eeLRkqU.png</normaloff>:/images/images/eeLRkqU.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">b</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="vertical_layout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTreeWidget" name="show_tree">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>230</number>
         </attribute>
         <attribute name="headerMinimumSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>#</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>FHD</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Subgroup</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
        </widget>
        <widget class="QTreeWidget" name="available_tree">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>New Episodes</string>
          </property>
         </column>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="button_layout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="addnew_button">
          <property name="text">
           <string>Add New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_button">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="scan_button">
          <property name="text">
           <string>Scan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="download_button">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Download</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="scan_progress">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
